lib/psql.js

```diff
@@ -11,11 +11,11 @@ function * exec (db, query) {
   debug(query)
   let env = Object.assign({}, process.env, {
```

Comment:
We should let `process.env` override these values, not the other way around, no?